spec.md

```diff
@@ -586,18 +589,68 @@ schema being used.
           "40 concurrent connections"
         ]
       }
+    }, {
+      "name": "fake-deprecated-plan",
+      "id": "009aba-XXXX-XXXX-XXXX-dace631cd648",
+      "description": "This is deprecated.",
+      "free": false,
+      "deprecation": {
+        "description": "Don't use this anymore.",
+        "since": "2018-04-20T17:00:00+00:00",
+        "eol": "2020-02-28T08:00:00+00:00",
+        "alternatives": [{
+            "description": "Use this plan instead.",
+            "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648"
+          }
+        ]
+      }
     }]
   }]
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| description | string | A short description of the deprecation. |
+| since | date string | Date and time when the Service Broker deprecated the service or plan. If specified, it is expected to be an ISO 8601 formatted date and time, with time zone. |
+| eol | date string | Date and time when the Service Broker expects to end-of-life (remove) the service or plan from the catalog. If specified, it is expected to be an ISO 8601 formatted date and time, with time zone. |
+| alternatives | array of [CatalogObjectId](#catalog-object-id-object) objects | An array of services or plans that the Service Broker considers to be good alternatives. |
```

Comment:
Can I suggest that this deprecation object should have an additional required field that enumerates which operations are (or aren't) still supported for the moment? It would be useful for platforms and end users alike to understand, for instance, that the service or plan in question can still be deprovisioned or unbound, but not provisioned or bound.

(I know this PR proposes binding MUST still be supported for a deprecated service or plan, but I have a comment to make about that as well.)

Reply:
I had original be thinking down this line but we (kibbles and I) had a debate and the winning factor was how deprecated is used in software, you can set your compile to throw errors if you use a deprecated method, but the API does not get to say. The platform can add restrictions, etc. But the brokers role is just to say "yo, this will go away soon"

What you want should happen after the broker has removed the plan or class from the catalog and eol has elapsed. 

So to be clear, deprecated means nothing has changed about this plan/class but change might happen soon, and the platform can help the user out by warning them.